maxim/logger/gemini/utils.py

```
@@ -49,10 +49,18 @@ def parse_chat_message(
-                        )
+                        {
+                            "role": role,
+                            "content": GeminiUtils.parse_part_text(m) or "",
+                        }
                     )
+        elif isinstance(message, str):
+            request_messages.append(
+                {
+                    "role": role,
+                    "content": message,
+                }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)